ivy.el

```diff
@@ -1515,7 +1519,7 @@ See variable `ivy-recursive-restore' for further information."
   (interactive)
   (ivy-call)
   (with-ivy-window
-    (recenter-top-bottom)))
```

Comment:
This function does not recenter exclusively but alternates between top, center,
 and bottom (which is not consistent with its docstring, I think)